src/concrete/ml/sklearn/tree_to_numpy.py

```diff
@@ -14,16 +15,16 @@
 )
 from ..onnx.convert import OPSET_VERSION_FOR_ONNX_EXPORT, get_equivalent_numpy_forward_from_onnx
 from ..onnx.onnx_model_manipulations import clean_graph_at_node_op_type, remove_node_types
+from ..onnx.onnx_utils import get_op_type
 from ..quantization import QuantizedArray
 from ..quantization.quantizers import UniformQuantizer
 
 # pylint: disable=wrong-import-position,wrong-import-order
 
 # Silence Hummingbird warnings
 warnings.filterwarnings("ignore")
-from hummingbird.ml import convert as hb_convert  # noqa: E402
 
-# pylint: enable=wrong-import-position,wrong-import-order
```

Comment:
I'm surprised pylint does not complain anymore

Reply:
We already have it in row 22. Why adding it twice?

Reply:
it's `enable` here, line 22 is `disable`, because we cannot do `disable-next` and it;s always better to enable right after we don't need the disable anymore (safer !)